src/stories/components/MainMenu/MainMenu.js

```diff
@@ -33,44 +32,31 @@ export const MainMenu = () => {
     }
   }, [currentUser]);
 
-  const handleLogout = () => {
-    // setError('');
-    logout()
-      .catch((error) => {
-        // setError(error);
-      });
-  };
-
   return (
     <ul className="main-menu">
+      { currentUser ?
+        <li className="main-menu__item">
+          Welcome back,&nbsp;
+          <strong>{profile ?
+            profile.username :
+            currentUser.fullName
```

Comment:
`currentUser` does not have a `fullName`. There is an attribute `displayName` but thats just null.
The profile always has a `username` and I left `email` in `else` just as a safety net